core/src/test/resources/spark-events-qualification/frequency-eventlog-set/empty_eventlog2

```diff
@@ -0,0 +1,7 @@
+{"Event":"SparkListenerLogStart","Spark Version":"3.1.1"}
```

Comment:
we are really trying to stay away from checking in more static event logs, if its something we can generate or copy that would be better.